README.md

```diff
@@ -137,3 +137,10 @@ ACCERT can be executed both through the Workbench interface by pressing `run` or
 `python ACCERT/src/Main.py -i myinput.son`
 
 
+
+# A New Tool is implemented in ACCERT to Quantify Capital Cost Reduction Pathways for Advanced Nuclear Reactors
+- A tool to showcase different pathways to reducing the capital cost of advanced reactor plants.
```

Comment:
A new tool has been developed to simulate the different pathways to reduce the costs of advanced nuclear power plants when constructed through large order books. This tool inputs various levers that represent high-level decision making and outputs the capital costs and construction duration starting from the first plant to the last plant in the order book.

Reply:
Done